src/test/java/ahmad/ParserTest.java

```diff
@@ -0,0 +1,88 @@
+package ahmad;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+import ahmad.exceptions.CommandInvalidException;
+
+public class ParserTest {
+
+    @Test
+    public void markTest() {
+        try {
+            Parser.parse("mark 1");
+        } catch (CommandInvalidException e) {
+            fail("Mark fail");
+        }
+    }
+
+    @Test
+    public void unMarkTest() {

```

Comment:
Rename method to "unmarkTest" to adhere to the camel case format for method names